cavern/README.md

```diff
@@ -57,6 +57,12 @@ org.opencadc.cavern.resourceID = ivo://{authority}/{name}
 # (optional) identify which container nodes are allocations
 org.opencadc.cavern.allocationParent = {top level node}
 
+# (optional) provide a class implementing the org.opencadc.cavern.nodes.QuotaPlugin interface to control Quotas
+# for users.
+# Optional, but the default of NoQuotaPlugin will get used if not specified, and users will have free reign.
+#
```

Comment:
need to document the choices in the text below

In fenwick and tantar we had plugins that are internal (artifactSelector and ResolutionPolicy respectively) which we started out putting fully qualified class names and then changed to a simple set of terms and mapped that to classes internally. I think for internally implemented (which I expect the quota plugins will be) that's slightly nicer and it gets rid of really dynamic plugin loading.

I'd probably follow the style in tantar.